moped-editor/src/views/dev/LookupsView/index.js

```diff
@@ -81,95 +81,100 @@ const LookupsView = () => {
 
   return (
     <ApolloErrorHandler error={error}>
-      <Page title="Moped Data Dictionary">
+      <Page title="Data Dictionary">
         <Container maxWidth="xl">
-          <Grid container spacing={3} sx={{ marginTop: 3 }}>
-            <Grid item xs={12}>
-              <Typography variant="h1">Moped Data Dictionary</Typography>
-            </Grid>
-          </Grid>
-          <Grid
-            container
-            spacing={3}
-            sx={{ marginTop: 3 }}
-            component={Paper}
-            ref={refs._scroll_to_top}
-          >
-            {SETTINGS.map((recordType) => (
-              <Grid item key={recordType.key} sx={{ marginBottom: 3 }}>
-                <Button
-                  color="primary"
-                  variant="outlined"
-                  component={Link}
-                  to={createRecordKeyHash(recordType.key)}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    scrollToTable(recordType.key, refs);
-                    history.replace(createRecordKeyHash(recordType.key));
-                  }}
-                >
-                  {recordType.label}
-                </Button>
-              </Grid>
-            ))}
-          </Grid>
-          {SETTINGS.map((recordType) => (
+          <Paper sx={{ paddingLeft: 3 }}>
             <Grid
               container
               spacing={3}
-              sx={{ marginTop: 3 }}
-              component={Paper}
-              key={recordType.key}
-              ref={refs[recordType.key]}
+              sx={{ marginTop: 3, scrollMarginTop: 24 }}
```

Comment:
I added scrollMarginTop so there is some buffer when scrolling to the very top from a lookup table. Previously, it would bring the lookup buttons into view with the dictionary title hidden behind the navbar.